app/assets/stylesheets/sections/deploy.scss

```diff
@@ -0,0 +1,56 @@
+$dropbox_color: #1081de;
```

Comment:
Name of variable `dropbox_color` should be written in all lowercase letters with hyphens instead of underscores